addons/sale_management/models/sale_order_option.py

```diff
@@ -48,6 +48,9 @@ class SaleOrderOption(models.Model):
         compute='_compute_price_unit',
         store=True, readonly=False,
         required=True, precompute=True)
+
+    is_price_set_manually = fields.Boolean(default=False)
```

Comment:
```suggestion
    is_price_set_manually = fields.Boolean(default=False)
```

There blank lines between field because of different kind of field so use can put this field closer to `price_unit` field or move it in technical field section